src/State/CritterDialog.cpp

```diff
@@ -134,6 +134,8 @@ void CritterDialog::setQuestion(const std::string& value)
 {
     auto question = getTextArea("question");
     question->setText(std::string("  ") + value);
+    question->setLineOffset(0);
+    Game::getInstance()->mouse()->setState(Input::Mouse::Cursor::BIG_ARROW);
```

Comment:
Purely aesthetics. If you are currently viewing a multi-scroll question, the mouse cursor will show as an up or down arrow to indicate this. If you then press a number on the keyboard and a single-scroll question appears, the cursor will not change, so I explicitly set it.

I just checked vanilla FO2 and found that this isn't original behavior, so I removed that line.